src/assets/custom/css/_sass/_services-page-intro.scss

```diff
@@ -1,23 +1,140 @@
+$header-background-nav-height--large-and-extra-large: $vertical-overlap * 4;
+$header-background-nav-image-height--large-and-extra-large: $vertical-overlap * 5;
+$header-background-nav-image-height--medium: 300px;
+$text-container-inner-margin-top: -$vertical-overlap * 2;
+
 .services-page-intro__image {
   background-color: #474b4a;
   background-position: center center;
   background-size: cover;
 
   @include small-and-medium {
-    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-small-medium.jpg");
-    height: 0;
-    padding-top: 200 / 360 * 100%;
+    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-small-medium-v2.jpg");  
   }
+
   @include large-and-extra-large {
-    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-large-extra-large.jpg");
-  }
-  @include large {
-    height: 0;
-    padding-top: 650 / 1820 * 100%;
+    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-large-extra-large-v2.jpg");
   }
+
   @include extra-large {
-    @include max-width--framed;
-    height: 465px;
+    height: $header-background-nav-image-height--large-and-extra-large;
+    margin-bottom: $vertical-overlap;
+  }
+}
+
+.services-page-intro__navigation {
+  @include max-width--out-there;
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+
+  @include small {
+    padding: 5% 0;
+  }
+
+  @include medium {
+    height: $header-background-nav-image-height--medium;
+  }
+
+  @include large-and-extra-large {
+    height: $header-background-nav-height--large-and-extra-large;
+  }
+}
+
+.services-page-intro__icon-container {
+  @include small {
+    align-items: center;
+    display: flex;
+    flex-basis: 50%;
+    flex-direction: column;
+    justify-content: center;
+    padding: 5% 5%;
+  }
+}
+
+$icon-image-height--large-extra-large: 150px;
+$icon-image-height--small-medium: 115px;
+$icon-image-width--large-extra-large: 130px;
+$icon-image-width--small-medium: 100px;
+
+.services-page-intro__navigation-link {
+  color: white;
+
+  & .service-line-icon {
+    height: 55%;
+    fill: white;
+  }
+
+  &:hover, &:focus {
+    text-decoration: none;
+    transition: color 0.3s ease;
+    color: $grey--light;
+
+    .service-line-icon {
+      transition: fill 0.3s ease;
+      fill: $grey--light;
+    }
+  }
+
+  @include small-and-medium {
+    width: $icon-image-width--small-medium;
+  }
+
+  @include large-and-extra-large {
+    width: $icon-image-width--large-extra-large;
+  }
+}
+
+.services-page-intro__icon--software-modernisation {
```

Comment:
If the icons were to be an include, this background image could be passed into each include. We'd not these in that case.